SCAutolib/models/file.py

```diff
@@ -53,6 +54,10 @@ def __init__(self, filepath: Union[str, Path], template: str = None):
         if template is not None:
             self._template = Path(template)
 
+    @property
+    def path(self):
```

Comment:
Do we want to add simple one-line docstring for purposes of felf-generated documentation?